validator/api/src/main/java/tech/pegasys/teku/validator/api/AttesterDuties.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.validator.api;
+
+import java.util.Objects;
+import tech.pegasys.teku.bls.BLSPublicKey;
+import tech.pegasys.teku.infrastructure.unsigned.UInt64;
+
+public class AttesterDuties {
+
+  private final BLSPublicKey publicKey;
+  private final int validatorIndex;
+  private final int committeeLength;
+  // the committee index of the committee that includes the specified validator
```

Comment:
nit: can we make these java doc comments:
```suggestion
  /** the committee index of the committee that includes the specified validator */
```

Mostly just because it shows up as an explanation in more places which makes it more useful.